### ISSUE TITLE
unable to execute commands in container

### ISSUE THREAD

**Issue by alfredbez** [ongr-io/development-environment#28]
```
$ lsb_release -d; uname -r; docker-compose --version; docker --version
Description:    Ubuntu 16.04.1 LTS
4.4.0-51-generic
docker-compose version 1.9.0, build 2585387
Docker version 1.12.3, build 6b644ec
```
```
$ docker-compose up -d
Starting ongrdocker_mariadb-data_1
Starting ongrdocker_npm_1
Starting ongrdocker_mariadb_1
Starting ongrdocker_fpm_1
Starting ongrdocker_nginx_1
$ docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                           NAMES
22270e64fa72        ongr/nginx          "nginx -g 'daemon off"   About an hour ago   Up 3 minutes        443/tcp, 0.0.0.0:8000->80/tcp   ongrdocker_nginx_1
28026980bd30        ongr/php-fpm        "php-fpm"                About an hour ago   Up 3 minutes        9000/tcp                        ongrdocker_fpm_1
7b0aaa00e1ea        mariadb:10.1.10     "/docker-entrypoint.s"   About an hour ago   Up 3 minutes        0.0.0.0:3300->3306/tcp          ongrdocker_mariadb_1
$ docker-compose exec php-fpm composer install
ERROR: No such service: php-fpm
$ docker-compose exec php-fpm bash
ERROR: No such service: php-fpm
```

**Comment by saimaz** [comment]
It has to be executed as `docker-compose exec fpm composer install`. The container is named as `fpm` in the docker compose file.

**Comment by alfredbez** [comment]
Fixed by 7a8beccb7be172e68c2e07c20b0b369af0507160
